hack/make.ps1

```diff
@@ -424,10 +425,17 @@ Try {
     # Run autogen if building binaries or running unit tests.
     if ($Client -or $Daemon -or $TestUnit) {
         Write-Host "INFO: Invoking autogen..."
-        Try { .\hack\make\.go-autogen.ps1 -CommitString $gitCommit -DockerVersion $dockerVersion -Platform "$env:PLATFORM" -Product "$env:PRODUCT" }
+        Try { .\hack\make\.go-autogen.ps1 -CommitString $gitCommit -DockerVersion $dockerVersion -Platform "$env:PLATFORM" -Product "$env:PRODUCT" -PackagerName "$env:PACKAGER_NAME" }
         Catch [Exception] { Throw $_ }
     }
```

Comment:
I realise it's not changing here, but it seems like `.go-autogen.ps1` only knows how to autogen for `$Daemon` case? Is there any point in calling it for the `$Client` or `$TestUnit` cases?

Reply:
Good point, I was also wondering the same thing. As of now there is no `autogen` build tag and everything is handle through ldflags it should be fine to only autogen for `$Daemon`. I will check that.